docs/api-publishing/08-ComplianceTesting.md

```diff
@@ -0,0 +1,132 @@
+
+### Key Benefits of Compliance Testing: 
+
+**Legal and regulatory adherence**: Ensures that the product or system complies with laws and regulations to avoid legal penalties. 
+
+**API Provider and API Consumer confidence**: Demonstrates that API consumption meets compliance requirements, which can help build confidence with API Providers, API Consumers and stakeholders.
+
+**Risk mitigation**: Identifies risks and vulnerabilities early in the development process, minimising the chances of security breaches, data leaks, privacy violations, or regulatory issues. 
+
+**Improved quality**: Enhances the overall services quality, security, and performance; improves data quality and data integrity by embedding the quality requirements into design and assurance process.
+
+## Key Considerations of Compliance Testing
+API Provide **SHOULD** take the following key considerations during the design of the compliance test:
+
+- Align Compliance Testing to business use cases, ensuring the process is easy to adopt 
+
+- Provide clear instructions and criteria (e.g. using [Gherkin syntax](https://cucumber.io/docs/gherkin/reference/)) 
+
+- Provide information of mandatory tests and conditions (e.g. a test is only required to complete if API Consumer has permission to view a patient's contact details)
+
+- Provide example or test data where feasible 
+
+- Address the **SPICE** concerns
+    - **S**ecurity - For example, 
+    - **P**rivacy - For example, [AsyncAPI Consumer](./04-AsyncAPI.md) discard the irrelevant events 
+    - **I**dentity - For example, API Consumer uses MFA to authenticate service users 
+    - **C**linical Safety - For example, the Patient immunisation information must be displayed in full regardless the screen resolution
+    - **E**quity - For example, 
+
+- Tests should reflect real life scenarios
+
+- Automate testing where it can
+
+- Tests can be extended with additional tests with incremental development 
+
+- Tests are reliable, repeatable, and traceable
+
+- Edge cases are considered (e.g. long names of patients, medicines, etc.) 
+
+- Remediation path is provided for test failure  
+
+- Recertification process is defined for periodic compliance testing and change of compliance status 
+
+- Provide examples and environment to demonstrate compliance (e.g. a compliance demo)
+
+- Provide test report or summary template  
+
+- Consider alternative or innovative tools to ease the compliance testing process, e.g. recorded demo, automation, report generation, etc. 
+
+## Compliance testing process 
+
+*API Provider* **MUST** provide compliant testing process definition. Diagrams **SHOULD** be used that describe an API Consumer compliance testing journey. 
+ 
+The process of compliance testing involves several steps to verify that the system meets the required compliance. Below is how a typical compliance test process looks like:
```

Comment:
Can you update the diagram to be clear on who is doing what? i.e. API consumer tasks vs provider tasks.

Reply:
Added actors. Pls review.